### DISCUSSION TITLE
Swagger/openapi

### DISCUSSION THREAD

**Discussion by RidgewellServices** [vercel/next.js#14338] [Help]
Has anyone found a swagger or openapi v3 package that integrates with Next.js server?
Or, have you implemented something on your own?

I'm using express with express router for my api routes.

Thanks for any help.

**Comment by ToeFungi** [comment]
Would also be useful to have a OpenAPI spec so that clients can be easily generated with the OpenAPI Generator. Anyone know if this exists?
